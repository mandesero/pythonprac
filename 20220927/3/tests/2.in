1,2,3
4,5,6
100,12,31
214,325,46
213,43,54
1534,43,466
