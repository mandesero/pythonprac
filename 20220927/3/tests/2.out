5242,540,1552
11125,1773,3250
71510,34349,19694
